project/src/main/java/com/google/sps/data/Event.java

```diff
@@ -0,0 +1,205 @@
+package com.google.sps.data;
+
+import java.lang.StringBuilder;
+import java.time.format.DateTimeFormatter;
+import java.time.LocalDate;
+import java.util.HashSet;
+import java.util.Random; 
+import java.util.Set;
+
+/** Class containing Event object 
+Setters for variables that user can change about event
+*/
+public final class Event {
+  private final int eventId = 1000000;
+  private String name;
+  private String description;
+  private Set<String> labels;
+  private String location;
+  private LocalDate date;
+  private Set<VolunteeringOpportunity> opportunities;
+  private Set<User> attendees;
+  private User host;
+
+  @Override
+  public String toString() { 
+    //TO DO: add VolunteeringOpp and User toString() based on those classes when pushed
+    StringBuilder str = new StringBuilder(); 
+    str.append("Event Info: \n");
+    str.append("Name: " + name + "\n");
+    str.append("Description: " + description + "\n");
+    str.append("Location: " + location + "\n");
+    str.append("Date: " + date.format(DateTimeFormatter.ofPattern("dd/MM/yyyy")) + "\n");
```

Comment:
Can use String.format here instead of concatenation.

Reply:
https://stackoverflow.com/questions/44117788/performance-between-string-format-and-stringbuilder 
I used StringBuilder() which is faster -- the cons vs String.format here say that sometimes Stringbuilder is unreadable but faster -- but i think readbility is fine in this case